jackson-jq/src/main/java/net/thisptr/jackson/jq/Scope.java

```diff
@@ -135,13 +135,12 @@ public ObjectMapper getObjectMapper() {
 		public List<JqFuncDef> functions = new ArrayList<>();
 	}
 
-	/**
-	 * Dynamically resolve the path for a resource as packages may be relocated, e.g. by
-	 * the maven-shade-plugin.
-	 */
-	private static final String resolvePath(final Class<?> clazz, final String name) {
```

Comment:
`resolvePath` is not needed: it's always called with the same parameters and the result could be computed statically once and for all.

Reply:
your call